app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsCardView.kt

```diff
@@ -19,11 +19,14 @@ class SuggestedEditsCardView(context: Context) : DefaultFeedCardView<SuggestedEd
     }
 
     private val binding = ViewSuggestedEditsCardBinding.inflate(LayoutInflater.from(context), this, true)
+    // TODO: to introduce a function to clear the fragmentMap when the card is refreshed.
+    private var fragmentMap = mutableMapOf<Int, List<Fragment>>()
```

Comment:
This `map` is for saving the created fragments and reusing them, otherwise new fragment will be created after this card view is visible.